tensorflow/workspace.bzl

```diff
@@ -76,7 +76,6 @@ def tf_workspace(path_prefix = "", tf_repo_name = ""):
   native.new_http_archive(
       name = "libxsmm_archive",
       urls = [
-          # "http://bazel-mirror.storage.googleapis.com/github.com/hfp/libxsmm/archive/1.6.1.tar.gz",
```

Comment:
There's no consequences to having this here and the URL not being active, aside from 500ms extra latency, which is amortized out by the eight downloader threads.

My process right now is I run that script you reviewed every few days or so, and whatever GCS URLs exist in the file that aren't currently mirrored, get mirrored.